core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -473,24 +510,56 @@ public static Timer timer(String name) {
    * The synchronized keyword is added for correctness with {@link #resetAllMetrics}
    */
   private static synchronized List<alluxio.grpc.Metric> reportMetrics(InstanceType instanceType) {
+    if (!sReported) {
+      initShouldReportMetrics(instanceType);
+      sReported = true;
+    }
     List<alluxio.grpc.Metric> rpcMetrics = new ArrayList<>(20);
```

Comment:
For worker, 12 metrics we are reporting. But it's not guaranteed since we have `Worker.BytesReadPerUfs.UFS:<UFS_ADDRESS> ` and `Worker.BytesReadPerUfs.UFS:<UFS_ADDRESS>` which may have multiple metrics to report.
For client,  5 metrics to report.